channel.json

```diff
@@ -68,6 +68,7 @@
 		"https://raw.githubusercontent.com/keeganstreet/sublime-specificity/master/packages.json",
 		"https://raw.githubusercontent.com/kik0220/sublimetext_japanize/master/packages.json",
 		"https://raw.githubusercontent.com/kylederkacz/lettuce-farmer/master/packages.json",
+		"https://raw.githubusercontent.com/Jun-Software/JunLang_package/package/package.json",
```

Comment:
Please remove this line, it's not needed. Just your repo listed in j.json is enough.